docs/source/notebooks/subset-user-input.ipynb

```diff
@@ -90,7 +90,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ds.heat_wave_frequency.plot()
</pre><p>I'm assuming the changing memory address is not picked up by the output diff during E2E tests? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/280/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>We have a regex to ignore those memory address change.</p><p> </p>